src/webpack/transform.ts

```diff
@@ -254,7 +259,12 @@ function addTransform(
         transpileOnly: getEnv() === Env.Dev && transformConfig.transpileOnlyWhenDev,
         compilerOptions,
         // 方便项目直接把内部依赖（portal-base / fe-core 等）的源码 link 进来一起构建调试
-        allowTsInNodeModules: true
+        allowTsInNodeModules: true,
+        compiler: transformConfig.useProjectTypeScript ?
+          require.resolve('typescript', {
+            paths: [abs('node_modules'), nodeModulesOfBuilder]
```

Comment:
从行为来看，这个选项更接近“prefer project TypeScript”，而不是“use project TypeScript”？

如果确实想要“use project TypeScript”的话，可能

```suggestion
            paths: [abs('node_modules')]
```

会更合适？

Reply:
done